spec/controllers/api/v1/comments_controller_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CommentsController, type: :controller do
+  let(:user) { create(:user, :with_task) }
+  let(:task) { user.tasks.first }
+  let(:params) {}
+
+  before { @request.headers[I18n.t('auth')] = JsonWebToken.encode(user_id: user.id) }
+
+  describe 'GET index' do
+    context 'when Success' do
```

Comment:
`describe 'Success' do` looks better

Reply:
Do you mean `context 'Success'`?

Reply:
No, I mean `describe`
Your test structure should have the following view
```
  describe 'GET index' do
    describe 'Success' do
      context 'with some conditions' do
        ...
      end
    end
end
```